pkg/client/status_reporter.go

```diff
@@ -27,10 +27,11 @@ import (
 )
 
 const (
-	unavailableMessage          string = "Rollout of the monitoring stack failed and is degraded. Please investigate the degraded status error."
-	asExpectedReason            string = "AsExpected"
-	StorageNotConfiguredMessage        = "Prometheus is running without persistent storage which can lead to data loss during upgrades and cluster disruptions. Please refer to the official documentation to see how to configure storage for Prometheus: https://docs.openshift.com/container-platform/4.8/monitoring/configuring-the-monitoring-stack.html"
-	StorageNotConfiguredReason         = "PrometheusDataPersistenceNotConfigured"
+	unavailableMessage              string = "Rollout of the monitoring stack failed and is degraded. Please investigate the degraded status error."
+	asExpectedReason                string = "AsExpected"
+	StorageNotConfiguredMessage            = "Prometheus is running without persistent storage which can lead to data loss during upgrades and cluster disruptions. Please refer to the official documentation to see how to configure storage for Prometheus: https://docs.openshift.com/container-platform/4.8/monitoring/configuring-the-monitoring-stack.html"
+	StorageNotConfiguredReason             = "PrometheusDataPersistenceNotConfigured"
+	WorkloadIncorrectlySpreadReason        = "WorkloadIncorrectlySpread"
```

Comment:
```suggestion
	WorkloadIncorrectlyBalancedReason        = "WorkloadIncorrectlyBalanced"
```
I suppose this is left over from changing the name further down?

Tbh I think `balanced` is also not quite the right naming here. This is probably nit-licking but I think names are important. Ultimately this comes down to the data not being replicated in a highly available way. Maybe at least for user facing messages something like `WorkloadDataSinglePointOfFailure` sends a clearer message? To me `IncorrecltyBalanced` lacks the immediate message that this can lead to data loss.

I can live with the `IncorrecltyBalanced` approach but I also think we can do better.

Reply:
Yeah I missed that one.

I think that's a good point you raised here, "balanced" isn't a very expressive term for the users whereas if we were to mention the problem of single point of failures which is very common in IT, it would make more sense to them. I will update the user-facing messages as you suggested.
If you have any renaming suggestions for `IncorrectlyBalanced`, I am happy to do the change, but I am personally out of inspiration on that one.